### ISSUE TITLE
[Camera] Failed to read exposureBiasesByMode dictionary

### ISSUE THREAD

**Issue by euthribeiro** [ivpusic/react-native-image-crop-picker#1424]
[Camera] Failed to read exposureBiasesByMode dictionary: Error Domain=NSCocoaErrorDomain Code=4864 "*** -[NSKeyedUnarchiver _initForReadingFromData:error:throwLegacyExceptions:]: data is NULL" UserInfo={NSDebugDescription=*** -[NSKeyedUnarchiver _initForReadingFromData:error:throwLegacyExceptions:]: data is NULL}

react-native-image-crop-picker: 0.35.
React Native: 0.63.2

**Comment by cjvilla2012** [comment]
I see the same problem. The result is no data returned from ImagePicker.openCamera. It appears to be an iOS 14 problem:

https://stackoverflow.com/questions/64061582/xcode-camera-failed-to-read-exposurebiasesbymode-dictionary

**Comment by euthribeiro** [comment]
I managed to solve the problem. In fact, it is not directly related to react-native-image-crop-picker. The problem was that I was using react-native-actionsheet to give the user the option to open the camera or the gallery. When I opened the react-native-actionsheet and pressed one of the options, the camera was superimposing the react-native-actionsheet (modal) which generated a conflict, because apparently in IOS it is not possible for one Modal to overlap the other.

So, to solve the problem, I defined a timeout so that it is possible to close the modal before opening the camera.
